apps/src/studio/Item.js

```diff
@@ -46,7 +49,7 @@ Item.__resetIds = function () {
 Item.prototype.createElement = function (parentElement) {
   // create our clipping path/rect
   this.clipPath = document.createElementNS(SVG_NS, 'clipPath');
-  var clipId = 'item_clippath_' + (uniqueId++);
+  var clipId = 'item_clippath_' + uniqueId;
```

Comment:
What if we have `var nextId = (uniqueId++);` at the top of this function, and then use `nextId` instead of `uniqueId` throughout. Might be a little less confusing than mutating uniqueId somewhere in the middle.